WooCommerce/Resources/Images.xcassets/tablet-success.imageset/Contents.json

```diff
@@ -0,0 +1,15 @@
+{
+  "images" : [
+    {
+      "filename" : "tablet-success-light.pdf",
```

Comment:
just curious, why is this drawing called "tablet-success"? I just couldn't guess what this is from the name 😅  also, do we have an equivalent image for dark mode?

Reply:
It was called that way on Figma, apparently what the lady is holding is a tablet.
And there isn't a dark version in the designs, but I'll make sure to [ask G if one is needed](https://github.com/woocommerce/woocommerce-ios/issues/2544#issuecomment-667105875).